fix(importer): support scoped packages

Alexander-Taran opened the pull request:
packages with names like @aspnet/signalr-client were treated like versioned

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/aurelia/cli?pullRequest=804) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aurelia/cli?pullRequest=804) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aurelia/cli?pullRequest=804) it.</sub>

zewa666 commented:
Great, so this PR should now properly support scoped packages. Do you have an example which you tested it with?

Alexander-Taran commented:
I was toying with `@aspnet/signalr-client`. 
And `au install` went perfectly well, but import failed. Because resolved package name was an empty string.
So I went on and ran cli with debugger and stepped through until I've found the culprit.
First thing that went wrong is the detection of a versioned package.
Whenever there is a @ symbol in package name  (even if it is the first symbol) - `hasVersion()` returns true. 
But changing only that didn't suffice because `parseVersioned()` would fail for versioned and scoped packages. So I changed the logic there to find the last occurrence of @ and split the string there.

Alexander-Taran commented:
Something else I can do to help merge this?

JeroenVinke commented:
If yoo could rename the commit to “fix(importer): support scoped packages” or something in that format, then it will show up in the changelog that is generated, and also in the release blog post. I’ll review it this week or next week, I appreciate your patience and contribution

Alexander-Taran commented:
done
